src/ui/components/results/universalresultscomponent.js

```diff
@@ -55,6 +56,9 @@ export default class UniversalResultsComponent extends Component {
     const sections = data.sections || [];
     const query = this.core.storage.get(StorageKeys.QUERY);
     const searchState = data.searchState || SearchStates.PRE_SEARCH;
+    ['preSearch', 'searchLoading', 'searchLoaded']
+      .forEach(state => this.removeContainerClass(`yxt-Results--${state}`));
```

Comment:
can we loop through the SearchStates and `getContainerClass` instead? that way the remove and add use the same class construction fn?